### DISCUSSION TITLE
Example ArtifactRequestMessage

### DISCUSSION THREAD

**Discussion by stefkoslowski** [International-Data-Spaces-Association/DataspaceConnector#67] [Q&A] [answered]
Hello, 

I am currently trying to receive an artifact via an ids multipart message sent over Postman.
I was wondering whether you could share a sample ArtifactRequestMessage including the required encoding and headers (HTTP and multipart). I am using v4.3.1.

**Comment by ghost** [comment, marked as answer]
Hello and sorry for the late reply, we have since added an example in the documentation for each IDS message that the DSC sends.

https://international-data-spaces-association.github.io/DataspaceConnector/Documentation/v6/Messages#artifactrequestmessage
